examples/microprofile/idcs/src/main/java/io/helidon/examples/microprofile/security/idcs/IdcsMain.java

```diff
@@ -35,13 +38,29 @@ private IdcsMain() {
      * @throws IOException when logging configuration fails
      */
     public static void main(String[] args) throws IOException {
-        LogManager.getLogManager().readConfiguration(IdcsMain.class.getResourceAsStream("/logging.properties"));
-        Server.create().start();
+        Server.builder()
+                .config(buildConfig())
+                .build()
+                .start();
 
         System.out.println("Endpoints:");
         System.out.println("Login");
         System.out.println("  http://localhost:7987/rest/login");
         System.out.println("Full security with scopes and roles (see IdcsResource.java)");
         System.out.println("  http://localhost:7987/rest/scopes");
+        System.out.println("A protected reactive service (see application.yaml - security.web-server)");
+        System.out.println("  http://localhost:7987/reactive");
+        System.out.println("A protected static resource (see application.yaml - security.web-server");
+        System.out.println("  http://localhost:7987/web/resource.html");
+    }
+
+    private static Config buildConfig() {
+        return Config.builder()
+                .sources(
+                        // you can use this file to override the defaults that are built-in
+                        file(System.getProperty("user.home") + "/helidon/conf/examples.yaml").optional(),
+                        // in jar file (see src/main/resources/application.yaml)
+                        classpath("application.yaml"))
+                .build();
```

Comment:
I think this is a very important feature, especially when the app will run from Kubernetes and might get an override config seeded. Probably you should feature this more prominently in other places as well and you might also change the default from a user home example to a more Kubernetes-ish one.

Reply:
We do plan to update our example to use meta configuration with description of how to override.
This is really for our developers - so we do not include our security setup specific URLs to the source base. 
There are examples specifically for configuration that show the capabilities. 
To use this in a k8s with config maps, the change is simple - just replace the path in the code with the file path the config map is mounted to...